src/signing/networkMessages/cosmos-hub-mainnet.js

```diff
@@ -15,39 +15,40 @@ export function SendTx(senderAddress, { to, amount }, network) {
 }
 
 // Staking
-export function StakeTx(senderAddress, { to, amount }, network) {
+export function StakeTx(senderAddress, { to, amount, denom }, network) {
```

Comment:
@Bitcoinera this is wrong. the schema for a StakeTx detail looks like this:

``` 
type StakeTx {
    amount: Coin!
    to: [String]!
  }
```

Reply:
But this is how you pass it in transactionData:

```
        return {
          type: messageType.STAKE,
          to: [this.targetValidator.operatorAddress],
          amount: this.amount,
          denom: this.stakingDenom
        }
```

Reply:
then passing the value from the actionmodal is wrong. sorry